src/fontra/views/editor/editor.js

```diff
@@ -776,6 +785,30 @@ export class EditorController {
     }
   }
 
+  initFallbackClipboardEventListeners() {
+    window.addEventListener("paste", async (event) => {
+      if (document.activeElement === this.canvasController.canvas) {
+        event.preventDefault();
+        this.doPaste();
+      }
+    });
+
+    window.addEventListener("copy", async (event) => {
+      if (document.activeElement === this.canvasController.canvas) {
+        event.preventDefault();
+        this._copyToEventClipboard(event);
+      }
+    });
+
+    window.addEventListener("cut", (event) => {
+      if (document.activeElement === this.canvasController.canvas) {
+        event.preventDefault();
+        this._copyToEventClipboard(event);
+        this.doCut(event);
```

Comment:
Instead of passing the event, let's just pass a flag. I think the flag argument coud be called "writeToClipboard", which should be `true` by default, so we'd pass `false` here.